source/tutorial/autocomplete-component.md

```diff
@@ -369,9 +369,9 @@ We will test the full logic of filtering in application tests, described in the
 
 Since our component is expecting the filter process to be asynchronous, we return promises from our filter, using [Ember's RSVP library](https://www.emberjs.com/api/ember/release/modules/rsvp).
 
-Next, we'll add the call to render the component to show the cities we've provided above.
+Next, we'll add the call to render the component to show the cities we've provided above. Since we use `await` when rendering, we must also mark the test's function as `async`.
```

Comment:
make the async and await words be hyperlinked:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await

Reply:
"Since we use [the `await` function](...) to wait for rendering, we must..."